traffic_control/clients/python/trafficops/tosession.py

```diff
@@ -2405,6 +2405,17 @@ def create_user(self, data=None):
 		:raises: Union[LoginError, OperationError]
 		"""
 
+	@api_request(u'put', u'users/{user_id:d}', (u'1.1', u'1.2', u'1.3',))
```

Comment:
you should add a `'1.4'` to this. Or maybe that should be done separately to _all_ the methods? idk, either way there ought to be an issue for adding it to the other methods that support 1.4 (or maybe just reworking it so you only need to specify a minimum version)

Reply:
done

Although, it appears to still be requesting 1.3. Off-hand, I don't know what the underlying mechanism is for which API version to request -- maybe the latest version common to all methods? Ideally we should only have to specify the lowest minor version that the method supports, but the client would make all API requests at the latest minor version (1.4) where possible.

Reply:
done

Although, it appears to still be requesting 1.3. Off-hand, I don't know what the underlying mechanism is for which API version to request -- maybe the latest version common to all methods? Ideally we should only have to specify the lowest minor version that the method supports, but the client would make all API requests at the latest minor version (1.4) where possible.

Reply:
The api version is set on the TOSession object at init, I think it currently defaults to `1.3`. It can probably be bumped to 1.4 at this point.